datasets/submission/preparation-guide.qmd

```diff
@@ -1,7 +1,16 @@
 ---
 title: Preparation guide
 ---
-Before you submit your data, please make sure
+
+This guide contains instructions on how to prepare your data for being uploaded to BigPicture.
+
+1.  [Check folder structure](#folder-structure)
+2.  [Verify xml file structure](#xmllint)
+
+
+
+## Check folder structure {#folder-structure}
+Before you submit your data, please make sure that your data structure complies with the structure below.
```

Comment:
```suggestion
Before you submit your data, please make sure that the folder structure of your dataset complies with the structure below.
```

"data structure" can mean different thing, maybe better to clarify it like this.